pkg/cmd/codespace/ssh.go

```diff
@@ -232,6 +271,7 @@ func (a *App) printOpenSSHConfig(ctx context.Context, opts sshOptions) (err erro
 			} else {
 				defer safeClose(session, &err)
 
+				// Pass empty key here because it doesn't actually connect to SSH
```

Comment:
I don't think we need this comment anymore, but if we do...

```suggestion
				// Fetch user information without starting a new SSH connection
```